src/m365/purview/commands/retentionevent/retentionevent-remove.spec.ts

```diff
@@ -161,7 +161,7 @@ describe(commands.RETENTIONEVENT_REMOVE, () => {
   });
 
   it('correctly handles random API error', async () => {
-    sinon.stub(request, 'delete').callsFake(() => Promise.reject('An error has occurred'));
+    sinon.stub(request, 'delete').callsFake(() => { throw 'An error has occurred'; });
```

Comment:
This refactor is sort of ok. In fact, we should try to use `.rejects(...)` here. But this won't work as simply with plain text.
You probably know I'm not a fan of writing `handles error` tests with plain text errors. As far as I know, not a single API returns a plain text error, so all these scenarios are not realistic at all.
In this case, I'd ask you to refactor the test and use a graph error object so we make the test relevant. In this case, we can also use `.rejects(...)` which makes it a bit cleaner.